tools/flat-storage/src/commands.rs

```diff
@@ -42,6 +43,17 @@ enum SubCommand {
 
     /// Run FlatState value inininig migration
     MigrateValueInlining(MigrateValueInliningCmd),
+
+    /// Construct and store trie in a separate directory from flat storage state for a given shard
+    ConstructTrieFromFlat(ConstructTriedFromFlatCmd),
+}
+
+#[derive(Parser)]
+pub struct ConstructTriedFromFlatCmd {
+    shard_id: u64,
+    /// Path to directory where the constructed trie would be stored. Note that there shouldn't be an
+    /// existing DB in the path provided.
+    write_store_path: PathBuf,
```

Comment:
For my education, what's the difference between Path and PathBuf? I see both are used in this file.

Reply:
Path is like str, PathBuf is like String.